README.md

```diff
@@ -474,6 +474,18 @@ _Options_
 > Returns `{"color": "red"}`.
 
 
+### `$pickset`
+
+Takes an array and a number `element` and returns a new n-element array
+containing unique values from the input array. If the number is larger than the
+length of the array, return `$missing` instead, which will remove the key from
+the resulting document.
+
+_Options_
+- `array` (required) Array of values or operators to choose from.
+- `element` (optional) The size of the output array. Default `1`.
```

Comment:
One small suggestion: `element` is not intuitive that this indicates the number of elements you want. Can we follow the chance.js notation and call it `quantity` instead?